pages/linux/ttyplot.md

```diff
@@ -0,0 +1,20 @@
+# ttyplot
+
+> A realtime plotting utility for terminal with data input from stdin / pipe.
+> More information: <https://github.com/tenox7/ttyplot>.
+
+- Plot the values `1`, `2` and `3` (`cat` prevents ttyplot to exit):
+
+`{ echo 1 2 3; cat } | ttyplot`
+
+- Set a specific title and unit:
+
+`{ echo 1 2 3; cat } | ttyplot -t {{title}} -u {{unit}}`
+
+- Use a while loop to continuously plot random values:
+
+`{ while true; do echo $RANDOM; sleep 1; done } | ttyplot`
+
+- Parse the output from other commands and visualize it:
```

Comment:
This specific regex probably only works for ping, so we shouldn't make the description too general
```suggestion
- Parse the output from `ping` and visualize it:
```